### PR TITLE
Fix rabbitmq_user when using force on user with permissions

### PR DESCRIPTION
**Pull request by sylr**
##### SUMMARY

On rabbitmq 3.7 using `force: yes` fails because outputs of rabbitmqctl gives an empty line

```
root@rabbitmq-vm1:~# rabbitmqctl -q list_user_permissions john

root@rabbitmq-vm1:~#
```

Provoking an error on

https://github.com/ansible/ansible/blob/456af458fcc6dfe29b89b8ab5a88ad1baea2fbe9/lib/ansible/modules/messaging/rabbitmq_user.py#L185

Because `perm.split('\t')` does not find any `\t`

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
module rabbitmq_user

### CODE REVIEW

File: lib/ansible/modules/messaging/rabbitmq_user.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def _exec(self, args, run_in_check_mode=False):
             if self.node is not None:
                 cmd.extend(['-n', self.node])
             rc, out, err = self.module.run_command(cmd + args, check_rc=True)
-            return out.splitlines()
```
Comment (webknjaz) [inline lib/ansible/modules/messaging/rabbitmq_user.py]: I think it's more like:
```python
return out.strip().splitlines()
```
which will have same result

Comment (sylr) [reply lib/ansible/modules/messaging/rabbitmq_user.py]: indeed

**Comment by sylr** [comment]
@webknjaz can we merge this ?

**Comment by sylr** [comment]
Anyone ? @dagwieers ?

**Comment by sylr** [comment]
@dagwieers @gundalow @webknjaz guys ?

**Comment by hyperized** [comment]
@sylr help me understand please. Do you have a playbook where this bug shows? What is the strip supposed to do exactly?

**Comment by dagwieers** [comment]
I am not involved in RabbitMQ. If you like to speed up RabbitMQ decision-making it could help to find other RabbitMQ contributors for discussing and merging RabbitMQ-related PRs. This works well for other technologies.

https://github.com/ansible/community/wiki/RabbitMQ

**Comment by webknjaz** [comment]
@sylr this is a change to the community-supported code. As Dag has outlined earlier, the decision is up to the maintainers and/or working group who is more knowledgeable in this area.
Here you can check who is on the related maintainers team: https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml#L262

**Comment by sylr** [comment]
@hyperized The strip remove the empty lines output by rabbitmqctl when you list permissions of a user which does not have any. This empty lines messes up the logic because it try to iterate on an empty line.

**Comment by hyperized** [comment]
@sylr any chance you can add a test to this code to confirm the change is effective?

**Comment by sylr** [comment]
I'm afraid I won't have time to do that sorry. But it works, we've been using this patch for a while now at our company.

**Pull request closed without merging**